server_addon/maya/server/settings/publishers.py

```diff
@@ -309,6 +386,166 @@ class ExtractAlembicModel(BaseSettingsModel):
+        title="Renderable Only",
+        description="Only export renderable visible shapes."
+    )
+    stripNamespaces: bool = SettingsField(
+        title="Strip Namespaces",
+        description=(
+            "Namespaces will be stripped off of the node before being written "
+            "to Alembic."
+        )
+    )
+    userAttr: str = SettingsField(
+        title="User Attr",
+        placeholder="attr1;attr2",
+        description=(
+            "Attributes matching by name will be included in the Alembic "
+            "export. Attributes should be separated by semi-colon `;`"
+        )
+    )
+    userAttrPrefix: str = SettingsField(
+        title="User Attr Prefix",
+        placeholder="prefix1;prefix2",
+        description=(
+            "Attributes starting with these prefixes will be included in the "
+            "Alembic export. Attributes should be separated by semi-colon `;`"
+        )
+    )
+    uvsOnly: bool = SettingsField(
+        title="UVs Only",
+        description=(
+            "If this flag is present, only uv data for PolyMesh and SubD "
+            "shapes will be written to the Alembic file."
+        )
+    )
+    uvWrite: bool = SettingsField(
+        title="UV Write",
+        description=(
+            "Uv data for PolyMesh and SubD shapes will be written to the "
+            "Alembic file."
+        )
+    )
+    verbose: bool = SettingsField(
+        title="Verbose",
+        description="Prints the current frame that is being evaluated."
+    )
+    visibleOnly: bool = SettingsField(
+        title="Visible Only",
+        description="Only export dag objects visible during frame range."
+    )
+    wholeFrameGeo: bool = SettingsField(
+        title="Whole Frame Geo",
+        description=(
+            "Data for geometry will only be written out on whole frames."
+        )
+    )
+    worldSpace: bool = SettingsField(
+        title="World Space",
+        description="Any root nodes will be stored in world space."
+    )
+    writeColorSets: bool = SettingsField(
+        title="Write Color Sets",
+        description="Write vertex colors with the geometry."
+    )
+    writeFaceSets: bool = SettingsField(
+        title="Write Face Sets",
+        description="Write face sets with the geometry."
+    )
+    writeNormals: bool = SettingsField(
+        title="Write Normals",
+        description="Write normals with the deforming geometry."
+    )
+    writeUVSets: bool = SettingsField(
+        title="Write UV Sets",
+        description=(
+            "Write all uv sets on MFnMeshes as vector 2 indexed geometry "
+            "parameters with face varying scope."
+        )
+    )
+    writeVisibility: bool = SettingsField(
+        title="Write Visibility",
+        description=(
+            "Visibility state will be stored in the Alembic file. Otherwise "
+            "everything written out is treated as visible."
+        )
+    )
+    overrides: list[str] = SettingsField(
```

Comment:
I wonder if this key `overrides` is too generic? Should it be `user_editable` or `user_configurable` or `editable_in_publisher`?

Reply:
Dont mind changing it to something else, just need a decision.

Reply:
@iLLiCiTiT @antirotor @mkolar any preference?

Reply:
I have no idea what is this used for so can't make correct opinion, but agree that `overrides` is too generic.